ibm_quantum_schemas/models/executor/version_0_1_0/models.py

```diff
@@ -0,0 +1,162 @@
+# This code is a Qiskit project.
+#
+# (C) Copyright IBM 2025.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Models"""
+
+from typing import Annotated, Literal, Self, Union
+
+from pydantic import BaseModel, Field, model_validator
+
+from ...base_params_model import BaseParamsModel
+from ...qpy_model import QpyModelV13ToV16
+from ...qubit_sparse_pauli_list_model import QubitSparsePauliListModel
+from ...samplex_model import SamplexModel
+from ...tensor_model import F64TensorModel, TensorModel
+
+
+class ParamsModel(BaseParamsModel):
+    """Schema version 1 of the inner parameters."""
+
+    schema_version: Literal["v0.1.0"] = "v0.1.0"
```

Comment:
This will spread out to  a bunch of places like folder names. Maybe you can have a think about whether it's better todo it later.